fastkafka/_application/app.py

```diff
@@ -579,6 +584,9 @@ async def _shutdown_consumers(
 ) -> None:
     if self._kafka_consumer_tasks:
         await asyncio.wait(self._kafka_consumer_tasks)
+        for task in self._kafka_consumer_tasks:
+            if self.test_mode and task.exception():
```

Comment:
@kumaranvpl 
Thanks for the PR.
I think we can move the if condition for test_mode before the for loop